fixtures/src/components/controlled/checkbox.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Checkbox } from 'carbon-react/lib/__experimental__/components/checkbox';
+import { LogConsumer } from '../log';
+
+const ControlledCheckbox = () => {
+  const [checked, setState] = useState(false);
```

Comment:
Naming convention when using the `useState()` hook:
```suggestion
  const [checked, setChecked] = useState(false);
```